teamA/jinwoo/controllers/postController.js

```diff
@@ -0,0 +1,51 @@
+const postService = require('../services/postService');
+
+const postUp = async (req, res) => {
+  try {
+    const { title, content, userId } = req.body;
+
+    if ( !title || !content || !userId ) {
+      return res.status(400).json({ message : 'KEY_ERROR' });
+    }
+
+    await postService.postUp( title, content, userId );
+    return res.status(201).json({ message : 'POSTUP_SUCCESS' });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message : err.message });
+  }
+};
+
+const postList = async (req, res) => {
+    const postsWesta = await postService.postList();
+    return res.status(201).json({ data : postsWesta });
```

Comment:
```suggestion
    return res.status(200).json({ data : postsWesta });
```
201 status code는 무언가 생성 되었을 때 주로 반환하는 상태코드입니다. 
원하는 정보(데이터)가 잘 반환되었다는 의미를 가지는 200으로 정정해주세요 :)